.docker/api-fetcher/main.js

```diff
@@ -0,0 +1,11 @@
+const { gotScraping } = require('got-scraping');
```

Comment:
I would like to avoid node.js dependencies in this repo. I'm sure there are Rust alternatives.